modules/custom/cp_station_list/cp_station_list.info.yml

```diff
@@ -4,4 +4,5 @@ package: CP
 type: module
 core_version_requirement: ^8.8 || ^9
 libraries:
-  - cp_station_list/station-list
\ No newline at end of file
+  - cp_station_list/station-list
+  - cp_station_list/stations
```

Comment:
It doesn't look like you added a `stations` library. The only library defined in this module is [station-list](https://github.com/ICOS-Carbon-Portal/drupal/blob/4314e0da84c9012985c08bfdeab58b4a9efea3e7/modules/custom/cp_station_list/cp_station_list.libraries.yml#L1).